bundler/lib/bundler/templates/newgem/github/workflows/main.yml.tt

```diff
@@ -1,6 +1,14 @@
 name: Ruby
 
-on: [push,pull_request]
+strategy:
+  matrix:
+    ruby: 3.0.0
+
+on:
+  push:
+    config[:git_default_branch]
```

Comment:
This should be an array including only the default branch, which you need to wrap with `<%= %>`. So

```suggestion
    - <%= config[:git_default_branch] %>
```